<launch>
  <node name = "slam_node" pkg = "ohm_tsd_slam" type = "slam_node" output = "screen">
      <param name = "laser_topic" type = "string" value = "/simon/scan"/>
      <param name = "x_off_factor" type = "double" value = "0.5"/>
      <param name = "y_off_factor" type = "double" value = "0.5"/>
      <param name = "cell_octave_factor" type = "int" value ="11" />
      <param name = "cellsize" type = "double" value = "0.025" />
      <param name = "range_filter" type = "bool" value = "true" />
      <param name = "min_range" type = "double" value = "0.0" />
      <param name = "max_range" type = "double" value = "20.0" />
      <param name = "low_reflectivity_range" type = "double" value = "5.0" />
      <param name = "occ_grid_time_interval" type = "double" value = "2.0" />
      <param name = "loop_rate" type = "double" value = "30.0" />
      <param name = "sensor_static_offset_x" type = "double" value = "0.0" />
      <param name = "pose_topic" type = "string" value = "pose" />
      <param name = "tf_base_frame" type = "string" value = "/map" />
      <param name = "tf_child_frame" type = "string" value = "base_footprint" />
      <param name = "truncation_radius" type = "double" value = "2.0" />
      <param name = "object_inflation_factor" type = "int" value = "0"/>
      <param name = "use_object_inflation" type = "bool" value = "false" />
      <param name = "robot_length" type = "double" value = "0.75" />
      <param name = "robot_width" type = "double" value = "0.5" /> 
      <param name = "use_icpsac" type = "bool" value = "true" />
      <param name = "dist_filter_max" type = "double" value = "0.5" />  
      <param name = "icp_iterations" type = "int" value = "30" />
  </node>
</launch>
